database/snapshotOptions/backup.json

```diff
@@ -0,0 +1,14 @@
+{
+  "filters": {},
+  "includeTables": [],
+  "excludeTables": [],
+  "shouldReInitialise": true,
+  "usePgDump": true,
+  "skipZip": true,
```

Comment:
Added this `skipZip` parameter. This means it won't make a zip file for the "takeSnapshot" step, as the backup script will be creating its own (encrypted) zip immediately after. With large snapshots and high compression levels, zipping twice could be unnecessarily expensive.